onyo/lib/commands.py

```diff
@@ -402,6 +403,8 @@ def onyo_edit(inventory: Inventory,
         Paths of assets to edit.
     message
         A custom commit message.
+    no_auto_message
+        If True, do not auto-generate a commit message subject line.
```

Comment:
I know that this is matching the CLI flag `--no-auto-message` (which I do believe is the right name). But I am not a fan of passing `True` to enable the disabling of something.

I would prefer seeing the default as `auto_message=True` and we pass `False` to disable the `auto_message`.

Argparse supports `store_false` as the `action` and then `dest` to name the variable (so that it differs from the flag name).